messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingSubsystemParser_6_0.java

```diff
@@ -161,6 +162,71 @@ public PersistentResourceXMLDescription getParserDescription(){
                                                         ConnectionFactoryAttributes.Pooled.INITIAL_CONNECT_ATTEMPTS,
                                                         ConnectionFactoryAttributes.Pooled.STATISTICS_ENABLED);
 
+        final PersistentResourceXMLBuilder externalPooledConnectionFactory =
```

Comment:
outside of the path element, is there any difference in the attributes exposed by the `external` variant of the pooled-connection-factory?

Reply:
default values and capabilities